tests/Spout/Reader/XLSX/Helper/CellValueFormatterTest.php

```diff
@@ -94,8 +94,8 @@ public function dataProviderForTestFormatNumericCellValueWithNumbers()
             [-42.5, -42.5, 'double'],
             ['42', 42, 'integer'],
             ['42.5', 42.5, 'double'],
-            [865640023012945, 865640023012945, 'integer'],
-            ['865640023012945', 865640023012945, 'integer'],
+            [865640023012945, 865640023012945, PHP_INT_SIZE < 8 ? 'double' : 'integer'],
```

Comment:
Can you make it more explicit by defining a variable at the top of this function and adding a comment?